TheModernIlluminati/TheModernIlluminati/Pages/SearchCountry.cshtml

```diff
@@ -5,69 +5,61 @@
     List<TheModernIlluminati.Models.Laureate>
       filteredLaureates = (List<TheModernIlluminati.Models.Laureate>
       )ViewData["filteredLaureate"];
-}     
-      
+}
 
-    <h1>Search by Country</h1>
 
-    <form action="" method="POST" style="margin-top: 50px;">
-        <div class="form-group">
-            <label style="font-size:20px;" class="control-label" for="CountrySearch"><strong>Enter Country</strong></label>
-            <input type="text" class="form-control" name="CountrySearch" id="CountrySearch" style="width:50%" />
-        </div>
-        <div class="form-group">
-            <input type="submit" value="Search for Laureates" class="btn btn-primary" />
-        </div>
-    </form>
+<h1>Search by Country</h1>
 
-    @if (Model.searchFinished == true)
-    {
-        <table class="table table-striped table-light">
-            <thead class="thead-dark">
-                <tr>
-                    <th>First Name</th>
-                    <th>Last Name</th>
-                    <th>Born City</th>
-                    <th>Born</th>
-                    <th>Birth Country</th>
-                    <th>Gender</th>
-                    <th>Year of Winning</th>
-                    <th>Category</th>
-                    <th>Share</th>
+<form action="" method="POST" style="margin-top: 50px;">
+    <div class="form-group">
+        <label style="font-size:20px;" class="control-label" for="CountrySearch"><strong>Enter Country</strong></label>
+        <input type="text" class="form-control" name="CountrySearch" id="CountrySearch" style="width:50%" />
+    </div>
+    <div class="form-group">
+        <input type="submit" value="Search for Laureates" class="btn btn-primary" />
+    </div>
+</form>
 
-                </tr>
-            </thead>
-            <tbody>
 
-                @foreach (var laureate in filteredLaureates)
-                {
-                    int i = 0;
+@if (filteredLaureates != null )
```

Comment:
removed searchFinished nullcheck and replaced it with filteredLaureates null check. searchFinished was an unnecessary Class property.
sources : https://docs.microsoft.com/en-us/visualstudio/ide/reference/unused-values-parameters?view=vs-2019